src/main/java/org/maveric/quarkus/panache/dtos/CustomerDto.java

```diff
@@ -0,0 +1,10 @@
+package org.maveric.quarkus.panache.dtos;
+
+import lombok.Data;
+
+@Data
+public class CustomerDto {
+
+    private Long customerId;
```

Comment:
What is the need for a DTO with just customerId? Cant we pass it as a parameter?

Reply:
Yes, i have changed